mesh/mesh.cpp

```diff
@@ -20,16 +20,18 @@
 #include "../general/tic_toc.hpp"
 #include "../general/gecko.hpp"
 #include "../general/kdtree.hpp"
+#include "../general/sets.hpp"
 #include "../fem/quadinterpolator.hpp"
 
-#include <iostream>
+// #include <iostream> // included by mesh.hpp
```

Comment:
Remove these?

Reply:
I was thinking of keeping these as reminders not to include some headers that are already included by `mesh.hpp`. I did that unintentionally and then had to revert the change.

However, if you think it is better to remove them, I can do that.

Maybe we can just replace them with a single comment like this:
```c++
// headers already included by mesh.hpp: <iostream>, <map>
```
What do you think?

Reply:
I also like the single comment.

Reply:
Okay. replaced with single comment in ceaf0af2c.